### PR TITLE
Added code owners

### PR DESCRIPTION
**Pull request by thesct22**

### CODE REVIEW

File: src/ansys/templates/python/osl_solution/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @asbfall @JoseRamonRodriguez @iazehaf @ansRMorin
```
[the remark this answers is not in the document]
Comment (thesct22) [reply src/ansys/templates/python/osl_solution/CODEOWNERS]: Done
But I dont have access to settings of this repo to give write access to the Solution-Maintainers group.
@asbfall will have to do that

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1 @@
+/src/ansys/templates/python/solution/ @ansys/Solution-Maintainers
```
Comment (nick-marnik) [inline CODEOWNERS:1]: Should there also be a line for `src/ansys/templates/python/osl_solution` ?

Comment (thesct22) [reply CODEOWNERS:1]: Me -> 🤡

Comment (thesct22) [reply CODEOWNERS:1]: Changes made

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+/src/ansys/templates/python/osl_solution @ansys/Solution-Maintainers
```
Comment (nick-marnik) [inline CODEOWNERS:2]: I **think** this is functionally equivalent to the line above, but the inconsistency with the trailing `/` is triggering me 😨 
```suggestion
/src/ansys/templates/python/osl_solution/ @ansys/Solution-Maintainers
```

**Comment by thesct22** [comment]
WazatozDogeGIF

**Review by nick-marnik — APPROVED** [review]
Happy Computer

**Pull request merged**